python/paddle/fluid/dygraph/dygraph_to_static/program_translator.py

```diff
@@ -88,15 +90,23 @@ def foo(x, y):
         # with decorator directly and function.__wrapped__ holds the actual function.
         func = getattr(func, '__wrapped__', func)
         source_code = func_to_source_code(func)
+
+        # TODO(liym27):
+        #  Consider this case: source_code in self._code_to_ast_caches,
+        #  but actually they are method in different class.
```

Comment:
NIT: English grammar, "they are methods in different classes".

Reply:
Thanks, done